modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAddressPermissions.testcase

```diff
@@ -0,0 +1,150 @@
+@component-name = "portal-commerce"
+definition {
+
+	property custom.properties = "jsonws.web.service.paths.excludes=${line.separator}company.security.strangers.verify=false";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property release.feature.flags.disable.DISABLE_PRIVATE_LAYOUTS = "true";
+	property testray.main.component.name = "Order Management";
+
+	setUp {
+		CommerceConfiguration.commerceSetUp();

```

Comment:
Change to CommerceConfiguration.commerceSetUp(minium = "true"); to create Minium site in setup.